hibernate-core/src/main/java/org/hibernate/stat/internal/StatisticsImpl.java

```diff
@@ -172,6 +175,9 @@ public void clear() {
 		deprecatedNaturalIdStatsMap.clear();
 
 		startTime = System.currentTimeMillis();
+
+		queryPlanCacheHitCount.reset();
+		queryPlanCacheMissCount.reset();
```

Comment:
It would be more consistent if these calls were before the `startTime` resolution.